README.md

```diff
@@ -21,14 +21,14 @@ Install it:
 npm i --save-dev eslint-doc-generator
 ```
 
-Add it as as script in `package.json` (included as a lint script to demonstrate how we can ensure it passes and is up-to-date on CI):
+Add it as as script in `package.json` (included as a lint script to demonstrate how we can ensure it passes and is up-to-date in CI):
 
 ```json
 {
   "scripts": {
     "lint": "npm-run-all \"lint:*\"",
     "lint:docs": "markdownlint \"**/*.md\"",
-    "lint:eslint-docs": "npm-run-all update:eslint-docs && git diff --exit-code",
+    "lint:eslint-docs": "npm run update:eslint-docs && git diff --exit-code",
```

Comment:
The goal of using `npm-run-all` here is that these scripts will all work with either npm, yarn, or any other package manager the user might be using.